diff -Naur dmz-cursor-theme-0.4.3.orig/DMZ-Black/index.theme dmz-cursor-theme-0.4.3/DMZ-Black/index.theme
--- dmz-cursor-theme-0.4.3.orig/DMZ-Black/index.theme	2007-05-24 23:24:00.000000000 +0300
+++ dmz-cursor-theme-0.4.3/DMZ-Black/index.theme	2011-01-24 21:53:16.836000049 +0200
@@ -1,6 +1,7 @@
 [Icon Theme]
 Name=DMZ (Black)
 Comment=Black accessible cursor theme
+Comment[tr]=Siyah renkli imleç teması
 Name[ar]=DMZ (أسود)
 Name[bg]=DMZ (Черно)
 Name[ca]=DMZ (Negre)
diff -Naur dmz-cursor-theme-0.4.3.orig/DMZ-White/index.theme dmz-cursor-theme-0.4.3/DMZ-White/index.theme
--- dmz-cursor-theme-0.4.3.orig/DMZ-White/index.theme	2007-05-24 23:24:00.000000000 +0300
+++ dmz-cursor-theme-0.4.3/DMZ-White/index.theme	2011-01-24 21:54:05.181000049 +0200
@@ -1,6 +1,7 @@
 [Icon Theme]
 Name=DMZ (White)
 Comment=White accessible cursor theme
+Comment[tr]=Beyaz renkli imleç teması
 Name[ar]=DMZ (أبيض)
 Name[bg]=DMZ (Бял цвят)
 Name[ca]=DMZ (Blanc)
