--- cmake/OpenFrameworks.cmake.orig	2011-01-03 18:49:23.000000000 +0200
+++ cmake/OpenFrameworks.cmake	2011-01-03 18:50:05.089000056 +0200
@@ -54,7 +54,6 @@
 macro(of_find_include includefile)
   find_path(${includefile}_INCPATH NAMES ${includefile} PATHS ${OPENFRAMEWORKS_INCLUDES_SEARCH_PATHS})
   if ( ${${includefile}_INCPATH} STREQUAL "${includefile}_INCPATH-NOTFOUND")
-    message(FATAL_ERROR "Cannot find ${includefile}")
   else()
     message(STATUS "Found ${includefile} in ${${includefile}_INCPATH}")
   endif()
@@ -75,7 +74,6 @@
   endif()
   
   if (NOT ${libname}_LIB)
-    message(FATAL_ERROR "Cannot find '${libname}' library")
   else()
     message(STATUS "Found '${libname}' library in ${${libname}_LIB}")
     set(OPENFRAMEWORKS_LIBRARIES ${OPENFRAMEWORKS_LIBRARIES} ${${libname}_LIB})
