--- cmake/OpenFrameworksConfig.cmake.orig	2010-10-09 19:57:28.000000000 +0300
+++ cmake/OpenFrameworksConfig.cmake	2011-01-03 18:35:40.723000054 +0200
@@ -4,17 +4,14 @@
 set(OPENFRAMEWORKS_ADDONS_ROOT "${OPENFRAMEWORKS_ROOT}/addons/")
 
 set(OPENFRAMEWORKS_INCLUDES_SEARCH_PATHS 
-                                 ${OPENFRAMEWORKS_ROOT}/libs/fmodex/include
-                                 ${OPENFRAMEWORKS_ROOT}/libs/FreeImage/include
-                                 ${OPENFRAMEWORKS_ROOT}/libs/freetype/include
-                                 ${OPENFRAMEWORKS_ROOT}/libs/glee/include
-                                 ${OPENFRAMEWORKS_ROOT}/libs/glu/include
-                                 ${OPENFRAMEWORKS_ROOT}/libs/glut/include
-                                 ${OPENFRAMEWORKS_ROOT}/libs/gstappsink/include
-                                 ${OPENFRAMEWORKS_ROOT}/libs/poco/include
+                                 /usr/include/fmodex
+                                 /usr/include
+                                 /usr/include/freetype2/freetype
+                                 /usr/include/GL
+                                 /usr/include/gstreamer-0.10/gst/app
+                                 /usr/include/Poco
                                  ${OPENFRAMEWORKS_ROOT}/libs/quicktime/include
-                                 ${OPENFRAMEWORKS_ROOT}/libs/rtAudio/include
-                                 ${OPENFRAMEWORKS_ROOT}/libs/unicap/include
+                                 /usr/include/unicap
                                  ${OPENFRAMEWORKS_ROOT}/libs/videoInput/include
                                  /usr/include/*
                                  
@@ -44,15 +41,8 @@
   endif()
 
      set(OPENFRAMEWORKS_LIBRARIES_SEARCH_PATHS 
-                                     ${OPENFRAMEWORKS_ROOT}/libs/fmodex/lib/${SUBLIBDIRSUFFIX}
-                                     ${OPENFRAMEWORKS_ROOT}/libs/FreeImage/lib/${SUBLIBDIRSUFFIX}
-                                     ${OPENFRAMEWORKS_ROOT}/libs/freetype/lib/${SUBLIBDIRSUFFIX}
-                                     ${OPENFRAMEWORKS_ROOT}/libs/glee/lib/${SUBLIBDIRSUFFIX}
-                                     ${OPENFRAMEWORKS_ROOT}/libs/glu/lib/${SUBLIBDIRSUFFIX}
-                                     ${OPENFRAMEWORKS_ROOT}/libs/glut/lib/${SUBLIBDIRSUFFIX}
-                                     ${OPENFRAMEWORKS_ROOT}/libs/gstappsink/lib/${SUBLIBDIRSUFFIX}
-                                     ${OPENFRAMEWORKS_ROOT}/libs/poco/lib/${SUBLIBDIRSUFFIX}
-                                     ${OPENFRAMEWORKS_ROOT}/libs/rtAudio/lib/${SUBLIBDIRSUFFIX}
+                                     /usr/lib
+                                     /usr/lib/gstreamer-0.10
                                      CACHE STRING "OpenFrameworks library search paths")
 endif() 
 
