--- autogen.sh.orig	2008-05-28 01:39:36.000000000 +0300
+++ autogen.sh	2011-10-31 14:12:07.541000019 +0200
@@ -1,4 +1,4 @@
-2#!/bin/sh
+#!/bin/sh
 
 # This script does all the magic calls to automake/autoconf and
 # friends that are needed to configure a cvs checkout.  You need a
@@ -13,7 +13,7 @@
 FILE=src/render.c
 
 AUTOCONF_REQUIRED_VERSION=2.54
-AUTOMAKE_REQUIRED_VERSION=1.6
+AUTOMAKE_REQUIRED_VERSION=1.11.1
 GLIB_REQUIRED_VERSION=2.0.0
 INTLTOOL_REQUIRED_VERSION=0.17
 
@@ -65,6 +65,9 @@
 elif (automake-1.6 --version) < /dev/null > /dev/null 2>&1; then
    AUTOMAKE=automake-1.6
    ACLOCAL=aclocal-1.6
+elif (automake-1.11 --version) < /dev/null > /dev/null 2>&1; then
+   AUTOMAKE=automake-1.11
+   ACLOCAL=aclocal-1.11
 else
     echo
     echo "  You must have automake 1.6 or newer installed to compile $PROJECT."
