--- data/screenkey.desktop.orig	2010-06-14 17:23:44.000000000 +0300
+++ data/screenkey.desktop	2011-02-07 02:01:36.590000020 +0200
@@ -1,10 +1,14 @@
 [Desktop Entry]
+Type=Application
+Version=1.0
 Encoding=UTF-8
 Name=Screenkey
+GenericName=Key Monitor
+GenericName[tr]=Tuş Gösterici
 Comment=Screencast your keys
-Version=0.2
+Comment[tr]=Tuşlarınızı gösterin
 Exec=screenkey
-Terminal=false
 Icon=preferences-desktop-keyboard-shortcuts
-Type=Application
-Categories=GNOME;Utility
+Terminal=false
+StartupNotify=true
+Categories=GNOME;Utility;
