diff -Naur refocus-it-2.0.0.orig/help/C/Makefile.am refocus-it-2.0.0/help/C/Makefile.am
--- refocus-it-2.0.0.orig/help/C/Makefile.am	2004-07-19 10:45:01.000000000 +0300
+++ refocus-it-2.0.0/help/C/Makefile.am	2011-10-31 22:00:41.723000013 +0200
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-helpdatadir = $(DATADIR)/help/C
+helpdatadir = $(DATADIR)/gimp-refocusit/help/C
 
 helpdata_DATA = \
         gimp-plugin-refocus-it.html	\
diff -Naur refocus-it-2.0.0.orig/help/C/Makefile.in refocus-it-2.0.0/help/C/Makefile.in
--- refocus-it-2.0.0.orig/help/C/Makefile.in	2004-07-19 13:54:33.000000000 +0300
+++ refocus-it-2.0.0/help/C/Makefile.in	2011-10-31 22:00:01.664000012 +0200
@@ -155,7 +155,7 @@
 sysconfdir = @sysconfdir@
 target_alias = @target_alias@
 
-helpdatadir = $(DATADIR)/help/C
+helpdatadir = $(DATADIR)/gimp-refocusit/help/C
 
 helpdata_DATA = \
         gimp-plugin-refocus-it.html	\
diff -Naur refocus-it-2.0.0.orig/help/images/Makefile.am refocus-it-2.0.0/help/images/Makefile.am
--- refocus-it-2.0.0.orig/help/images/Makefile.am	2003-11-29 14:08:51.000000000 +0200
+++ refocus-it-2.0.0/help/images/Makefile.am	2011-10-31 22:00:50.018000012 +0200
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-helpdatadir = $(DATADIR)/help/images
+helpdatadir = $(DATADIR)/gimp-refocusit/help/images
 
 helpdata_DATA = \
 	wilber.png
diff -Naur refocus-it-2.0.0.orig/help/images/Makefile.in refocus-it-2.0.0/help/images/Makefile.in
--- refocus-it-2.0.0.orig/help/images/Makefile.in	2004-07-19 13:54:33.000000000 +0300
+++ refocus-it-2.0.0/help/images/Makefile.in	2011-10-31 22:00:18.198000013 +0200
@@ -155,7 +155,7 @@
 sysconfdir = @sysconfdir@
 target_alias = @target_alias@
 
-helpdatadir = $(DATADIR)/help/images
+helpdatadir = $(DATADIR)/gimp-refocusit/help/images
 
 helpdata_DATA = \
 	wilber.png
diff -Naur refocus-it-2.0.0.orig/help/Makefile.am refocus-it-2.0.0/help/Makefile.am
--- refocus-it-2.0.0.orig/help/Makefile.am	2003-11-29 14:08:12.000000000 +0200
+++ refocus-it-2.0.0/help/Makefile.am	2011-10-31 22:00:31.353000013 +0200
@@ -2,7 +2,7 @@
 
 SUBDIRS = C images 
 
-helpdatadir = $(DATADIR)/help
+helpdatadir = $(DATADIR)/gimp-refocusit/help
 
 helpdata_DATA = gimp-help.xml
 
diff -Naur refocus-it-2.0.0.orig/help/Makefile.in refocus-it-2.0.0/help/Makefile.in
--- refocus-it-2.0.0.orig/help/Makefile.in	2004-07-19 13:54:33.000000000 +0300
+++ refocus-it-2.0.0/help/Makefile.in	2011-10-31 21:59:13.516000013 +0200
@@ -157,7 +157,7 @@
 
 SUBDIRS = C images 
 
-helpdatadir = $(DATADIR)/help
+helpdatadir = $(DATADIR)/gimp-refocusit/help
 
 helpdata_DATA = gimp-help.xml
 
