--- Makefile.orig	2011-06-29 05:53:54.000000000 +0300
+++ Makefile	2011-07-31 02:00:17.952000077 +0300
@@ -297,9 +297,8 @@
 	@find . -name "*.cmd" -exec rm '{}' \;
 
 install::
-	install -o root -g root -m 755 -d $(INSTALLDIR)
-	install -o root -g root -m 744 emgd.ko $(INSTALLDIR)
-	/sbin/depmod -a
+	install -o root -g root -m 755 -d $(DESTDIR)$(INSTALLDIR)
+	install -o root -g root -m 744 emgd.ko $(DESTDIR)$(INSTALLDIR)
 
 uninstall::
 	rmmod $(INSTALLDIR)/emgd.ko
