--- pysteg.py.orig	2011-03-14 02:39:55.000000000 +0200
+++ pysteg.py	2011-11-24 14:12:11.825000008 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2.6
+#!/usr/bin/env python
 #-*- encoding: utf-8 -*-
 
 
@@ -21,7 +21,7 @@
 
 builder = gtk.Builder()
 builder.add_from_file('%s/data/main.ui' % basepath)
-config = ConfigObj("%s/data/pySteg.conf" % basepath)
+config = ConfigObj("%s/.pySteg.conf" % os.path.expanduser('~'))
 # Check if steghide is installed
 if os.path.isfile("/usr/bin/steghide"):
     pass
