--- xournal.desktop.orig	2009-08-31 04:51:24.000000000 +0300
+++ xournal.desktop	2011-02-06 18:50:08.729000011 +0200
@@ -1,17 +1,17 @@
 [Desktop Entry]
 Encoding=UTF-8
 Type=Application
-
+Version=1.0
 Name=Xournal
+GenericName=Notetaker and PDF Annotator
+GenericName[tr]=Not Alıcı ve PDF İşaretleyici
 Comment=Take handwritten notes
-Name[fr]=Xournal
-Comment[fr]=Prise de notes manuscrites
-Name[ca]=Xournal
 Comment[ca]=Preneu notes a mà
-
+Comment[fr]=Prise de notes manuscrites
+Comment[tr]=El yazınızla notlar alın
 Exec=xournal %f
+Icon=xournal
 Terminal=false
 StartupNotify=true
 MimeType=application/x-xoj;application/pdf;
-Icon=xournal
 Categories=GNOME;GTK;Utility;TextEditor;
