--- po/Makefile.orig	2009-01-20 12:03:29.000000000 +0200
+++ po/Makefile	2011-10-31 22:29:03.894000005 +0200
@@ -24,7 +24,8 @@
 
 install: $(LANGUAGES)
 	for L in $(LANGUAGES); \
-	do install -v -m 0644 $$L.mo "$(LOCALEDIR)/$$L/LC_MESSAGES/gimp20-$(PLUGIN)-plug-in.mo"; \
+	do mkdir -p "$(DESTDIR)/$(LOCALEDIR)/$$L/LC_MESSAGES"; \
+	install -v -m 0644 $$L.mo "$(DESTDIR)/$(LOCALEDIR)/$$L/LC_MESSAGES/gimp20-$(PLUGIN)-plug-in.mo"; \
 done
 
 uninstall: $(LANGUAGES)
