--- configure.orig	2011-11-28 15:57:28.000000000 +0200
+++ configure	2011-11-29 13:27:25.747000016 +0200
@@ -5115,13 +5115,13 @@
 
 fi
 
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for cvSobel in -lcv" >&5
-$as_echo_n "checking for cvSobel in -lcv... " >&6; }
+{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for cvSobel in -lopencv_core -lopencv_imgproc -lopencv_highgui -lopencv_ml -lopencv_video -lopencv_features2d -lopencv_calib3d -lopencv_objdetect -lopencv_contrib -lopencv_legacy -lopencv_flann" >&5
+$as_echo_n "checking for cvSobel in -lopencv_core -lopencv_imgproc -lopencv_highgui -lopencv_ml -lopencv_video -lopencv_features2d -lopencv_calib3d -lopencv_objdetect -lopencv_contrib -lopencv_legacy -lopencv_flann... " >&6; }
 if ${ac_cv_lib_cv_cvSobel+:} false; then :
   $as_echo_n "(cached) " >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-lcv  $LIBS"
+LIBS="-lopencv_core -lopencv_imgproc -lopencv_highgui -lopencv_ml -lopencv_video -lopencv_features2d -lopencv_calib3d -lopencv_objdetect -lopencv_contrib -lopencv_legacy -lopencv_flann  $LIBS"
 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 
@@ -5156,7 +5156,7 @@
 #define HAVE_LIBCV 1
 _ACEOF
 
-  LIBS="-lcv $LIBS"
+  LIBS="-lopencv_core -lopencv_imgproc -lopencv_highgui -lopencv_ml -lopencv_video -lopencv_features2d -lopencv_calib3d -lopencv_objdetect -lopencv_contrib -lopencv_legacy -lopencv_flann $LIBS"
 
 else
 
@@ -5167,13 +5167,13 @@
 
 fi
 
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for main in -lhighgui" >&5
-$as_echo_n "checking for main in -lhighgui... " >&6; }
+{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for main in -lopencv_core -lopencv_imgproc -lopencv_highgui -lopencv_ml -lopencv_video -lopencv_features2d -lopencv_calib3d -lopencv_objdetect -lopencv_contrib -lopencv_legacy -lopencv_flann" >&5
+$as_echo_n "checking for main in -lopencv_core -lopencv_imgproc -lopencv_highgui -lopencv_ml -lopencv_video -lopencv_features2d -lopencv_calib3d -lopencv_objdetect -lopencv_contrib -lopencv_legacy -lopencv_flann... " >&6; }
 if ${ac_cv_lib_highgui_main+:} false; then :
   $as_echo_n "(cached) " >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-lhighgui  $LIBS"
+LIBS="-lopencv_core -lopencv_imgproc -lopencv_highgui -lopencv_ml -lopencv_video -lopencv_features2d -lopencv_calib3d -lopencv_objdetect -lopencv_contrib -lopencv_legacy -lopencv_flann  $LIBS"
 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 
@@ -5202,7 +5202,7 @@
 #define HAVE_LIBHIGHGUI 1
 _ACEOF
 
-  LIBS="-lhighgui $LIBS"
+  LIBS="-lopencv_core -lopencv_imgproc -lopencv_highgui -lopencv_ml -lopencv_video -lopencv_features2d -lopencv_calib3d -lopencv_objdetect -lopencv_contrib -lopencv_legacy -lopencv_flann $LIBS"
 
 else
 
