--- Makefile.am.orig	2002-04-01 04:17:17.000000000 +0300
+++ Makefile.am	2010-02-22 16:27:30.912660705 +0200
@@ -9,30 +9,30 @@
 
 install-data-local:
 	@if test -d /usr/share/consolefonts; then \
-	    echo " Installing matrix fonts in /usr/share/consolefonts..."; \
-	    $(INSTALL_DATA) $(srcdir)/matrix.fnt /usr/share/consolefonts; \
-	    $(INSTALL_DATA) $(srcdir)/matrix.psf.gz /usr/share/consolefonts; \
+	    echo " Installing matrix fonts in $(DESTDIR)/usr/share/consolefonts..."; \
+	    $(INSTALL_DATA) $(srcdir)/matrix.fnt $(DESTDIR)/usr/share/consolefonts; \
+	    $(INSTALL_DATA) $(srcdir)/matrix.psf.gz $(DESTDIR)/usr/share/consolefonts; \
 	fi
 	@if test -d /usr/lib/kbd/consolefonts; then \
-	    echo " Installing matrix fonts in /usr/lib/kbd/consolefonts..."; \
-	    $(INSTALL_DATA) $(srcdir)/matrix.fnt /usr/lib/kbd/consolefonts; \
-	    $(INSTALL_DATA) $(srcdir)/matrix.psf.gz /usr/lib/kbd/consolefonts; \
+	    echo " Installing matrix fonts in $(DESTDIR)/usr/lib/kbd/consolefonts..."; \
+	    $(INSTALL_DATA) $(srcdir)/matrix.fnt $(DESTDIR)/usr/lib/kbd/consolefonts; \
+	    $(INSTALL_DATA) $(srcdir)/matrix.psf.gz $(DESTDIR)/usr/lib/kbd/consolefonts; \
 	fi
 	@if test -d /usr/lib/X11/fonts/misc; then \
-	    echo " Installing X window matrix fonts in /usr/lib/X11/fonts/misc..."; \
-	    $(INSTALL_DATA) $(srcdir)/mtx.pcf /usr/lib/X11/fonts/misc; \
-	    $(INSTALL_DATA) $(srcdir)/mtx.pcf /usr/lib/X11/fonts/misc; \
+	    echo " Installing X window matrix fonts in $(DESTDIR)/usr/lib/X11/fonts/misc..."; \
+	    $(INSTALL_DATA) $(srcdir)/mtx.pcf $(DESTDIR)/usr/share/fonts/misc; \
+	    $(INSTALL_DATA) $(srcdir)/mtx.pcf $(DESTDIR)/usr/share/fonts/misc; \
 	    echo " Running mkfontdir /usr/lib/X11/fonts/misc..."; \
 	    $(MKFONTDIR) /usr/lib/X11/fonts/misc; \
 	    echo " Done.  If this is the first time you have installed CMatrix you will"; \
 	    echo " probably have to restart X window in order to use the mtx.pcf font."; \
 	else \
 	if test -d /usr/X11R6/lib/X11/fonts/misc; then \
-	    echo " Installing X window matrix fonts in /usr/X11R6/lib/X11/fonts/misc..."; \
-	    $(INSTALL_DATA) $(srcdir)/mtx.pcf /usr/X11R6/lib/X11/fonts/misc; \
-	    $(INSTALL_DATA) $(srcdir)/mtx.pcf /usr/X11R6/lib/X11/fonts/misc; \
-	    echo " Running mkfontdir /usr/X11R6/lib/X11/fonts/misc..."; \
-	    $(MKFONTDIR) /usr/X11R6/lib/X11/fonts/misc; \
+	    echo " Installing X window matrix fonts in $(DESTDIR)/usr/X11R6/lib/X11/fonts/misc..."; \
+	    $(INSTALL_DATA) $(srcdir)/mtx.pcf $(DESTDIR)/usr/X11R6/lib/X11/fonts/misc; \
+	    $(INSTALL_DATA) $(srcdir)/mtx.pcf $(DESTDIR)/usr/X11R6/lib/X11/fonts/misc; \
+	    echo " Running mkfontdir $(DESTDIR)/usr/X11R6/lib/X11/fonts/misc..."; \
+	    $(MKFONTDIR) $(DESTDIR)/usr/X11R6/lib/X11/fonts/misc; \
 	    echo " Done.  If this is the first time you have installed CMatrix you will"; \
 	    echo " probably have to restart X window in order to use the mtx.pcf font."; \
 	fi \
