diff -urpN gimp-gap-2.6.0.orig/gap/gap_story_dialog.h gimp-gap-2.6.0/gap/gap_story_dialog.h
--- gimp-gap-2.6.0.orig/gap/gap_story_dialog.h	2009-06-04 14:38:03.000000000 -0600
+++ gimp-gap-2.6.0/gap/gap_story_dialog.h	2009-07-10 13:30:35.833743151 -0600
@@ -32,6 +32,8 @@
 #include "gap_story_main.h"
 #include "gap_story_properties.h"
 
+GtkWidget * p_gtk_button_new_from_stock_icon(const char *stock_id);
+
 void    gap_storyboard_dialog(GapStbMainGlobalParams *gpp);
 
 void    gap_story_dlg_attw_render_all(GapStbAttrWidget *attw);
diff -urpN gimp-gap-2.6.0.orig/gap/gap_story_properties.c gimp-gap-2.6.0/gap/gap_story_properties.c
--- gimp-gap-2.6.0.orig/gap/gap_story_properties.c	2009-06-04 14:38:03.000000000 -0600
+++ gimp-gap-2.6.0/gap/gap_story_properties.c	2009-07-10 13:31:59.424583981 -0600
@@ -55,6 +55,7 @@
 #include "gap_timeconv.h"
 #include "gap_thumbnail.h"
 #include "gap_fmac_base.h"
+#include "gap_fmac_name.h"
 #include "gap_story_vthumb.h"
 
 
diff -urpN gimp-gap-2.6.0.orig/libgapvidutil/gap_gve_misc_util.c gimp-gap-2.6.0/libgapvidutil/gap_gve_misc_util.c
--- gimp-gap-2.6.0.orig/libgapvidutil/gap_gve_misc_util.c	2009-06-04 14:38:03.000000000 -0600
+++ gimp-gap-2.6.0/libgapvidutil/gap_gve_misc_util.c	2009-07-10 13:33:22.379426254 -0600
@@ -34,6 +34,7 @@
 #include <sys/stat.h>
 #include <errno.h>
 
+#include <glib/gstdio.h>
 
 /* GIMP includes */
 #include "gtk/gtk.h"
