Index: jpeglib.h
===================================================================
--- jpeglib.h.orig
+++ jpeglib.h
@@ -13,6 +13,10 @@
 #ifndef JPEGLIB_H
 #define JPEGLIB_H
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+
 /*
  * First we include the configuration files that record how this
  * installation of the JPEG library is set up.  jconfig.h can be
@@ -1093,4 +1097,8 @@ struct jpeg_color_quantizer { long dummy
 #include "jerror.h"		/* fetch error codes too */
 #endif
 
+#ifdef __cplusplus
+} /* extern "C" */
+#endif
+
 #endif /* JPEGLIB_H */
