diff -Naur src.orig/MsgWindow.cc src/MsgWindow.cc
--- src.orig/MsgWindow.cc	2005-05-01 17:08:37.000000000 +0300
+++ src/MsgWindow.cc	2011-11-24 14:47:03.580000008 +0200
@@ -23,7 +23,8 @@
 // This class implements a generic message window that can display any message 
 // as a popup
 
-#include "MsgWindow.h"
+#include "MsgWindow.h" 
+#include <cstdlib>
 
 // Message Window constructor
 MsgWindow::MsgWindow(int w, int h, std::string title, std::string text, std::string symbol)
